CMakeLists.txt

```diff
@@ -26,16 +26,19 @@ if(BUILD_TESTING)
   ament_lint_auto_find_test_dependencies()
 endif()
 
-add_executable(sensor src/sensor.cpp)
-ament_target_dependencies(sensor rclcpp geometry_msgs)
-install(TARGETS sensor
-  DESTINATION lib/${PROJECT_NAME})
+#add_executable(sensor src/sensor.cpp)
+#ament_target_dependencies(sensor rclcpp geometry_msgs)
+#install(TARGETS sensor
+#  DESTINATION lib/${PROJECT_NAME})
 
 add_executable(image src/image_publisher.cpp)
+ament_target_dependencies(image rclcpp sensor_msgs)
+install(TARGETS image
+  DESTINATION lib/${PROJECT_NAME})
 
-add_executable(controller src/controller.cpp)
-ament_target_dependencies(controller rclcpp geometry_msgs)
-install(TARGETS controller
-	DESTINATION lib/${PROJECT_NAME})
+#add_executable(controller src/controller.cpp)
+#ament_target_dependencies(controller rclcpp geometry_msgs)
+#install(TARGETS controller
+#	DESTINATION lib/${PROJECT_NAME})
```

Comment:
Could you remove these comments? (this problems are solved in master)

Reply:
I removed these line